### PR TITLE
Impl #314 GA Event Tracking on data details page

### LINKED ISSUE #314: Integrate GA library for tracking download events
The new **google analytics javascript library** could be integrated into the wpckan plugin (or even better into the wp-odm_theme theme in order to be accessible by all the plugins) and used for tracking dataset download events. Concretelly, following events should be tracked:

- When the dataset detail page is loaded, an event **dataset_view** would be sent to GAnalytics containing the id of the dataset in question.

- When the user clicks on the download button for a dataset's resource, an event **resource_download** would be sent to GAnalytics  containing the id of the dataset and the id of the resource.

- OPTIONAL: When the user user clicks on a related dataset listed on the sidebar, an event **related_dataset_link** would be sent to GAnalytics containing the id of the dataset the user is being linked to.

- [ ] Integrate GA javascript library (optimally via composer)
- [ ] Create additional js file under /js/ which contains javascript code

### PR DESCRIPTION
**Pull request by DragonVirus**
Fix #314

### CODE REVIEW

File: templates/dataset-detail.php [hunk 1]
```diff
@@ -195,3 +195,9 @@ function render_metadata_table($supported_fields,$data){
 }
 
 ?>
+<script>
+  //Send GA Event
+  jQuery(function(){
+    ga('send', 'event', 'Dataset', 'view', '<?php echo $dataset_id; ?>');
```
Comment (acorbi) [inline templates/dataset-detail.php]: @DragonVirus I do not see the GA library being added anywhere? Is this something you forgot and you might still need to add via bower or composer? Or is it already added via the Google Analytics dashboard plugin already installed?

Comment (DragonVirus) [inline templates/dataset-detail.php]: GA analytic JS is already included by GA plug-in we are using. So i think we don't need to include it again. 
I can't test it properly from local. Is it possible to implement it on PP and test it ?

Comment (acorbi) [inline templates/dataset-detail.php]: > i GA analytic JS is already included by GA plug-in we are using. So i think we don't need to include it again.

oki doki

> I can't test it properly from local. Is it possible to implement it on PP and test it ?

Sure! please update the version number and I'll merge it

Comment (DragonVirus) [inline templates/dataset-detail.php]: Done !

**Pull request merged**